src/Graphoscope/Measures/LongestPath.fs

```diff
@@ -0,0 +1,189 @@
+namespace Graphoscope.Measures
+
+open Graphoscope
+open System.Collections.Generic
+
+type LongestPath() =
```

Comment:
Yes, that's correct. But this is implemented because every type annotation in this library includes this constructor. It's simply designed to conform to the established schema.